app/components/node/details/node-details.component.ts

```diff
@@ -23,18 +25,32 @@ export class NodeDetailsComponent implements OnInit, OnDestroy {
     public nodeId: string;
     public poolId: string;
     public data: RxEntityProxy<NodeParams, Node>;
+    public poolData: RxEntityProxy<NodeParams, Pool>;
     public node: Node;
+    public pool: Pool;
 
     private _paramsSubscribers: Subscription[] = [];
 
-    constructor(private route: ActivatedRoute, private nodeService: NodeService, private fileService: FileService) {
+    constructor(
+        private route: ActivatedRoute,
+        private nodeService: NodeService,
+        private poolService: PoolService,
+        private fileService: FileService,
+        private sidebarManager: SidebarManager) {
         this.data = nodeService.get(null, null, {});
```

Comment:
yeah if the formatter was a bit smartter I would align those params with the first one instead of being just indentented on the next line.